cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaClusterTest.java

```diff
@@ -302,6 +302,30 @@ public void testMetricsConfigMap() {
         }
     }
 
+    @ParallelTest
+    public void testStrimziMetricsConfigs() {
```

Comment:
Rename to `testStrimziMetricsConfigs`.

Reply:
@fvaleri It is called `testStrimziMetricsConfigs`??

Reply:
Right, I may have overlooked this :D

Reply:
`testStrimziMetricsReporterConfigs` is what I meant. Sorry.

Reply:
Renamed